src/controller.py

```diff
@@ -73,7 +75,16 @@ def handle_request_nonce(connection_id, content):
 def handle_challenge_response(connection_id, content):
     logger.info("handle_attestation_challenge")
 
+    attestation_object = content.get("attestation_object")
     platform = content.get("platform")
+    app_version = content.get("app_version")
+    os_version_parts = content.get("os_version").split(" ")
```

Comment:
Any chance the first part of the os_version could have an additional space in it? Like "Android Oreo 9.1"?

Reply:
We're going to have to learn what each platform does. So far its' just `Android 14.4` so no, not that I know of but who knows what LG et al do.